lib/includes/Coding.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * A coding representation.  This should only be used in the context of a Concept.
+ */
+export default class Coding {
+  constructor(code, system, display) {
```

Comment:
I was surprised when I saw these using the constructor since none of the generated classes do and instead use `fromJSON`/field setters, but then I saw the Reference class does the same thing. I don't think it's worth changing now (unless you think it can be done really quickly & easily), but at some point I think we may want to try to make the Concept/Coding/Reference classes work the same way as the generated classes.

Reply:
We can look into it.  It's a little tricky for `concept` since the JSON passed in might be a `code`, a `Coding`, or a `Concept` -- but I think I already have to figure that out elsewhere anyway, so I guess it would be just moving that logic into here.

I think I was just thinking that from the point of view of a user, creating instances in code (not from JSON), having a simple constructor is easier than having to use a default constructor and then set those things individually.